Library/Formula/chicken.rb

```diff
@@ -17,12 +17,12 @@ def install
     ENV.deparallelize
 
     args = %W[
-      PLATFORM=macosx
+      PLATFORM=#{OS.mac? ? "macosx" : "linux"}
```

Comment:
Could you please change `"linux"` to `OS::NAME`? Sorry I didn't catch this before.